support/cas-server-support-thymeleaf/src/main/resources/templates/fragments/pwdupdateform.html

```diff
@@ -53,6 +53,32 @@
                 <span th:each="err : ${#fields.errors('*')}" th:utext="${err}">Error text</span>
             </div>
 
+            <div id="currentPasswordInputForm"
```

Comment:
Id here feels wrong; you're not using a form here...`currentPasswordPanel` or something similar should be enough I think.